tools/serverpod_cli/test/generator/dart/client_code_generator/class_field_test.dart

```diff
@@ -52,17 +80,32 @@ void main() {
       config: config,
     );
 
-    test('then a client-side class is generated with that class variable.', () {
-      var expectedFileName = path.join(
-        '..',
-        'example_project_client',
-        'lib',
-        'src',
-        'protocol',
-        'example.dart',
-      );
+    var compilationUnit = parseString(content: codeMap[expectedFileName]!).unit;
+    var maybeClassNamedExample = CompilationUnitHelpers.tryFindClassDeclaration(
+        compilationUnit,
+        name: testClassName);
+
+    group('then class named $testClassName', () {
+      var exampleClass = maybeClassNamedExample!;
+      test('has field in constructor.', () {
+        expect(
+            CompilationUnitHelpers.hasConstructorDeclaration(exampleClass,
+                name: null, parameters: ['this.title']),
```

Comment:
Question: Will this guarantee that no `required` keyword exists in the constructor? I.E. does this test fail if the required keyword is present?